task/backend/storetest/logstoretest.go

```diff
@@ -183,11 +183,23 @@ func listRunsTest(t *testing.T, crf CreateRunStoreFunc, drf DestroyRunStoreFunc)
 	ctx := context.Background()
 	ctx = pcontext.SetAuthorizer(ctx, makeNewAuthorization(ctx, t, makeAuthz))
 
-	if _, err := reader.ListRuns(ctx, task.ID, platform.RunFilter{Task: task.ID}); err == nil {
-		t.Fatal("with bad org ID, expected error but got nil")
+	{
+		r, err := reader.ListRuns(ctx, task.ID, platform.RunFilter{Task: task.ID})
+		if err != nil {
```

Comment:
I'm not totally sure about expecting a nil error here given a bad org ID. In particular, I'm surprised that a higher level test with a real query and storage engine didn't reject it since the org doesn't exist. Maybe this is working this way because of (temporary) special handling on the system bucket.

Can you add a comment here saying that this might fail in the future when system bucket authorization is checked more aggressively? Hopefully that will save some time for whoever deals with that.